src/content/docs/en/guides/media.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Use a DAM with Astro
+description: How to use a DAM to add images and videos to Astro
+i18nReady: true
+---
+import MediaGuidesNav from '~/components/MediaGuidesNav.astro';
+import ReadMore from '~/components/ReadMore.astro';
+import Badge from "~/components/Badge.astro"
+
+**Ready to connect a Headless DAM to your Astro project?** Follow one of our guides to integrate a hosted media system.
+
+:::tip
+Find [community-maintained integrations](https://astro.build/integrations/?search=cms) for connecting a DAM or hosted media system to your project in our integrations directory. 
+:::
+
+## Hosted Media Guides
+
+Note that many of these pages are **stubs**: they're collections of resources waiting for your contribution!
+
+<MediaGuidesNav />
+
+## Why use a DAM or hosted media?
+
+Using a DAM, or Digital Asset Manager, helps individual, teams, and organizations manage their image and video assets from a central location much like a [CMS](/en/guides/cms).
+
+The difference is the type of content being managed, where a DAM would be primarily made up of images, videos, other media assets like 3D models, and any metadata associated with those assets.
+
+This can be useful particularly when using a single source of truth for your assets between multiple web or mobile properties, if you're part of an organization that requires multiple teams to utilize the assets, or integrating into other content systems like a PIM (Product Information Manager) to connect your assets to products.
```

Comment:
```suggestion
This can be useful particularly when using a single source of truth for your assets between multiple web or mobile properties. This is important if you're part of an organization that requires multiple teams to use the same assets, or are integrating into other content systems like a PIM (Product Information Manager) to connect your assets to products.
```